threads task two.py

```diff
@@ -0,0 +1,66 @@
+"""
+Threads competition
+"""
+import time
+import threading
+
+lst_names = []
+lst_score = []
+event = threading.Event()
+
+
+def get_score(threads_name: str):
+    """
+    According to the event, the threads accessing this method try
+    to take a score from a global list and put it in another global list
+
+    :param threads_name: The name of the thread accessing this method.
+    """
+    event.wait()
+    if lst_score:
+        lst_score.pop(0)
+        lst_names.append(threads_name)
+        event.clear()
+        event.wait()
```

Comment:
This logic is complex. Explain it to me.

Reply:
after fixing the code I wrote this:
event.wait()
queue.get()
lst_names.append(threads_name)
event.clear()
The logic is that all threads have to wait because I don't start them at the same time. Then, when the event becomes True they get the score from a queue and put their name in a list so that I will know which thread succeeded. Then I change the event to False so that no other threads will put their name in the list of winners.